lib/services/trustedcluster.go

```diff
@@ -406,27 +406,30 @@ func (c *TrustedClusterV2) SetReverseTunnelAddress(e string) {
 // CanChangeState checks if the state change is allowed or not. If not, returns
 // an error explaining the reason.
 func (c *TrustedClusterV2) CanChangeStateTo(t TrustedCluster) error {
+	immutableFieldErr := func(name string) error {
+		return trace.BadParameter("can not update %s for existing trusted cluster, delete and re-create this trusted cluster with updated %s", name, name)
```

Comment:
We recently moved to using 'root/leaf' to remove confusion around which cluster shares trust and how they are connected.  I would suggest 

`can not update %s for existing leaf cluster, delete and re-create this leaf cluster with updated %s

Reply:
Yeah, this would be a great start towards using the terminology in code too.

Reply:
Yes, we should do this. Approved with this condition!